### DISCUSSION TITLE
Run as ... (Administrator), Locked Applikation

### DISCUSSION THREAD

**Discussion by drscream** [HotCakeX/Harden-Windows-Security#172] [Q&A] [answered]
I've applied the latest script from you and first of all thanks a lot for all your work! Also it might be important that I'm not a Windows User at all.

I've one "Administrator" account and one "User" account, based on the recommended setup. I would like to run and install programs in the user account with using "Run as ..." UI tooling. But I always receive the error that the Application is locked by the administrator.

I expect this is part of the Smart Control or WDAC? Is there an easy way to disable this or the an option to allow the "Run as ..." tooling? I tried to re-run your script (as Administrator) but it only closes the powershell terminal without any changes or questions.

**Comment by HotCakeX** [comment, marked as answer]
Hi,
Currently elevating to Admin privileges from standard user accounts isn't recommended for security reasons, [however this is going to change.](https://youtu.be/8T6ClX-y2AE?si=dIaaGg-pYf-RrEkQ&t=1295)
So for now best practices suggest either logging out of standard account and logging into Admin account or hot switching to Admin account, to do administrative tasks.

The 'Run as' interface doesn't work because of the policies applied by Microsoft security baselines.

You can change this policy and set it to prompting instead of denying

<img width="1280" alt="image" src="https://github.com/HotCakeX/Harden-Windows-Security/assets/118815227/9560c9e6-cd4a-4bf8-ba83-91bb55031ee9">

**Reply by drscream** [reply]
Thanks a lot for the details. I was not aware that this is part of Microsoft security baselines. Your suggested change works, thanks!

**Reply by HotCakeX** [reply]
You're welcome, glad i could help!
